src/pybind/mgr/cephadm/module.py

```diff
@@ -1764,11 +1764,14 @@ def enter_host_maintenance(self, hostname: str, force: bool = False) -> str:
 
         :raises OrchestratorError: Hostname is invalid, host is already in maintenance
         """
-        if len(self.cache.get_hosts()) == 1:
+        if yes_i_really_mean_it and not force:
+            raise OrchestratorError("--force must be passed with --yes_i_really_mean_it")
```

Comment:
Nit: I know you can type it as either `--yes-i-really-mean-it` or `--yes_i_really_mean_it`. I'm not sure what's more common "in the wild." But I usually think the former looks nicer. YYMV

Reply:
For switches I personally like hypens, for Ceph options I prefer underscores.  They can use spaces too, and I *think* hyphens but that might not be documented.

Reply:
switched to hyphens since this is just a bool variable here